src/CosmosDB/CosmosDB/Helpers/Constants.cs

```diff
@@ -190,5 +190,30 @@ internal static class Constants
+        public const string ManagedCassandraDatacenterNameHelpMessage = "Managed Cassandra Datacenter Name.";
+        public const string ManagedCassandraCassandraVersionHelpMessage = "The version of Cassandra chosen.";
+        public const string ManagedCassandraDatacenterObjectHelpMessage = "Managed Cassandra Datacenter object";
+        public const string ManagedCassandraDatacenterLocationHelpMessage = "Azure Location of the DataCenter.";
+        public const string ManagedCassandraHoursBetweenBackupsHelpMessage = "The number of hours between backup attempts.";
+        public const string ManagedCassandraExternalSeedNodesHelpMessage = "A list of ip addresses of the seed nodes of on-premise data centers.";
+        public const string ManagedCassandraClientCertificatesHelpMessage = "If specified, enables client certificate authentication to the Cassandra API.";        
+        public const string ManagedCassandraNodeCountHelpMessage = "The number of Cassandra virtual machines in this data center. The minimum value is 3.";
+        public const string ManagedCassandraExternalGossipCertificatesHelpMessage = "A list of certificates that the managed cassandra data center's should accept.";
+        public const string ManagedCassandraInitialCassandraAdminPasswordHelpMessage = "The intial password to be configured when a cluster is created for AuthenticationMethod Cassandra.";
+        public const string ManagedCassandraBase64EncodedCassandraYamlFragment = "This is a Base64 encoded yaml file that is a subset of cassandra.yaml. Supported fields will be honored and others will be ignored.";
+        public const string ManagedCassandraDataCenterDelegatedSubnetIdHelpMessage = "The resource id of a subnet where ip addresses of the Cassandra virtual machines will be allocated. This must be in the same region as datacenter location.";
+        public const string ManagedCassandraAuthenticationMethodHelpMessage = "Authentication mode can be None or Cassandra. If None, no authentication will be required to connect to the Cassandra API. If Cassandra, then passwords will be used.";
+        public const string ManagedCassandraRestoreFromBackupIdHelpMessage = "The resource id of a backup. If provided on create, the backup will be used to prepopulate the cluster. The cluster data center count and node counts must match the backup.";
+        public const string ManagedCassandraDelegatedSubnetIdHelpMessage = "The resource id of a subnet where the ip address of the cassandra management server will be allocated. This subnet must have connectivity to the DelegatedSubnetId subnet of each data center.";        
+        public const string ManagedCassandraClusterNameOverrideHelpMessage = "If a cluster must have a name that is not a valid azure resource name, this field can be specified to choose the Cassandra cluster name. Otherwise, the resource name will be used as the cluster name.";
```

Comment:
"By default, the Azure resource name is used to populate the clusterName field in cassandra.yaml. If this field is set, its value is used instead of the Azure resource name, for example, if you need a clusterName that is not a valid Azure resource name, or you need multiple clusters with the same clusterName.

Reply:
Thanks.